gtk4/src/auto/gesture.rs

```diff
@@ -68,20 +68,32 @@ pub trait GestureExt: 'static {
 
     fn get_property_n_points(&self) -> u32;
 
-    fn connect_begin<F: Fn(&Self, &gdk::EventSequence) + 'static>(&self, f: F) -> SignalHandlerId;
+    fn connect_begin<F: Fn(&Self, Option<&gdk::EventSequence>) + 'static>(
```

Comment:
This and below fixes https://github.com/gtk-rs/gtk4/issues/29

Reply:
What I mean here: add a `Fixes https://github.com/gtk-rs/gtk4/issues/29` to the commit message :)